src/hotspot/share/utilities/vmError.cpp

```diff
@@ -104,8 +104,8 @@ int               VMError::_lineno;
 size_t            VMError::_size;
 const size_t      VMError::_reattempt_required_stack_headroom = 64 * K;
 const intptr_t    VMError::segfault_address = pd_segfault_address;
-volatile intptr_t VMError::handshakeTimedOutThread = p2i(nullptr);
-volatile intptr_t VMError::safepointTimedOutThread = p2i(nullptr);
+volatile intptr_t VMError::_handshake_timeout_thread = p2i(nullptr);
+volatile intptr_t VMError::_safepoint_timeout_thread = p2i(nullptr);
```

Comment:
Sorry there was a typo in my suggestion - it should be `timed_out` to match the function names. (Or chang the functions to use `timedout` - I tend to use `timedout` as a single "word" in these contexts.